tests/rptest/services/redpanda.py

```diff
@@ -1078,6 +1081,14 @@ def raise_on_bad_logs(self, allow_list=None):
 
         bad_lines = collections.defaultdict(list)
         for node in self.nodes:
+
+            if self._skip_if_no_redpanda_log and not node.account.exists(
```

Comment:
Should we skip it or should we check a particular error message?

Reply:
The error message "/var/lib/redpanda/redpanda.log: No such file or directory" comes from bash not from the redpanda.log file so `redpanda.search_log()` is not applicable here.

Sure I could look for the error message from the grep call on line 1096 but, in my mind, that would result in skipping the log regardless.

Reply:
Yeah, this is fine. It's merged into the PR, thanks